isso/css/isso.css

```diff
@@ -159,6 +159,13 @@
     display: none;
 }
 
+.isso-target {
+    animation: target-fade 5s ease-out;
+}
+@keyframes target-fade {
+    0% { background-color: #eee5a1; }
```

Comment:
This color (light yellow) is quite jarring on dark background. Try e.g. the demo with `body { background: black; color: white; }`

Also, maybe add a little code comment such as `Flash comment light yellow when focussed`, either here or in embed.js